README.md

```diff
@@ -38,6 +38,14 @@ Use `--help` option in order to get the full list of options
 --help        Show this message and exit.
 ```
 
+## Usage with srt-live-transmit
+As mentioned this tool is intended for usage with [srt-live-transmit](https://github.com/Haivision/srt/blob/master/docs/apps/srt-live-transmit.md), for which you explicitly need to enable the export of statistics in .csv format. This can be done by passing the following parameters:
+- ` -statspf:csv` to ensure the CSV format is used when exporting.
+- ` -statsout:/path/to/desired/folder/filename-rcv.csv` to define where to write the statistics
+- ` -stats-report-frequency:1000` to define how often to write statistics (in ms), this example sets it to once per second.
```

Comment:
Not exactly. In `srt-live-transmit` the frequency is measured in the number of packets. See the description of the option "The frequency of SRT statistics collection, based on the number of packets." If you specify 1000, it means the statistics is calculated each 1000 packets.

To collect the statistics each N ms, please transition to [srt-xtransmit](https://github.com/maxsharabayko/srt-xtransmit) which is more powerful and handy for data analysis I believe. You will find the examples in README. There are 2 options for this `--statsfile stats-snd.csv --statsfreq 100ms`.

If you have time and interest, you can add the bit about `srt-xtransmit` here as well.

Reply:
Resolved in https://github.com/koenkarsten/srt-stats-plotting/commit/179fc9b02a77a96af7d18433b05490f72f8f7d57